crates/topos-sequencer-subnet-runtime/src/lib.rs

```diff
@@ -153,6 +160,27 @@ impl SubnetRuntimeProxyWorker {
     }
 }
 
+pub fn derive_endpoints(endpoint: &str) -> Result<(String, String), Error> {
+    let http_endpoint: String;
+    let ws_endpoint: String;
+
+    if endpoint.starts_with("http") {
```

Comment:
Isn't this the wrong way around? if `endpoint` is "https://example.com", `endpoint.starts_with("http")` will still return true yeah? I think the cases need to be flipped.

Reply:
You are right